core/src/test/scala/org/apache/spark/status/api/v1/SimpleDateParamTest.scala

```diff
@@ -22,7 +22,7 @@ class SimpleDateParamTest extends FunSuite with Matchers {
 
   test("date parsing") {
     new SimpleDateParam("2015-02-20T23:21:17.190GMT").timestamp should be (1424474477190L)
-    new SimpleDateParam("2015-02-20T17:21:17.190CST").timestamp should be (1424474477190L)
+    new SimpleDateParam("2015-02-20T17:21:17.190MDT").timestamp should be (1424474477190L)
```

Comment:
Ah sorry, it wasn't your PR; I mean this PR is making a similar but different change: https://github.com/apache/spark/pull/6377  We'll probably pull that in shortly anyway